tests/main_test.py

```diff
@@ -39,7 +39,7 @@ def test_get(self):
         :return:
         """
         response = requests.get(self.base_url + '/test_get')
-        cond = (response.content == b'Smart Ledger API Endpoint: OK')
+        cond = (response.content == b'{"msg":"Smart Ledger API Endpoint: OK"}\n')
```

Comment:
Probably safer to grab just the "msg" object from the dictionary and compare it against "Smart Ledger API Endpoint: OK" as a normal string rather than having everything as a binary string with the \n

Reply:
fixed